configs/sst_platform_tools-performance-tools-unwanted.yaml

```diff
@@ -0,0 +1,18 @@
+document: feedback-pipeline-workload
```

Comment:
Can you please change this to:

```
document: feedback-pipeline-unwanted
```

The exclusion list is a different document type.

Reply:
I updated the document type.

Reply:
Thank you!